src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Color {
+    WHITE,
+    RED,
+    BLACK,
+    BLUE,
+    GREEN,
+    BROWN,
+    YELLOW,
+}
```

Comment:
Remember to use the name() method when you need the String representation of these enum constants, instead of toString() or String.valueOf().